stroke.py

```diff
@@ -0,0 +1,27 @@
+import requests
+import re
+import os
+import time
+
+def get_stroke(hanzi):
+  return '&nbsp;'.join(map(get_stroke_single, hanzi))
```

Comment:
why `&nbsp;` 😣

Reply:
Good question. I want a single space between images because it's pretty, but a literal space will probably work.

Reply:
yah in HTML a since space is the same as `&nbsp;`

Reply:
What? Then why would `&nbsp;` exist? Or do you just mean they're the same visually, mod breaking rules?

Reply:
you need it if you want two or more spaces in a row...